src/gluonts/torch/modules/loss.py

```diff
@@ -48,10 +49,31 @@ def forward(
 
 
 class NegativeLogLikelihood(DistributionLoss):
+    def __init__(self, beta: float = 0.0):
+        """
+        Compute the negative log likelihood loss.
+
+        Parameters
+        ----------
+        beta
+            beta parameter from the paper: "On the Pitfalls of Heteroscedastic
```

Comment:
Maybe you can suggest the range (0, 1) for this hyperparameter, and reasonable values: 0 recovers NLL, 0.5 appears to be a reasonable choice (at least from the results in the paper)